edit/harvesterhci.io.virtualmachineimage.vue

```diff
@@ -80,6 +87,30 @@ export default {
 
       this.value.spec.url = url;
       if (VM_IMAGE_FILE_FORMAT.includes(fileSuffiic)) {
+        if (fileSuffiic) {
```

Comment:
fileSuffiic won't be null becauseof `VM_IMAGE_FILE_FORMAT.includes(fileSuffiic)`